en/flying/basic_flying.md

```diff
@@ -26,16 +26,20 @@ To arm the drone:
 <span id="takeoff-and-landing"></span>
 ## Takeoff
 
-The easiest way to takeoff is to use the automatic [Takeoff mode](../flight_modes/takeoff.md) (remembering that you need to [arm the vehicle](#arm) before you can engage the vehicle motors).
+The easiest way to takeoff (after [arming the vehicle](#arm)) is to use the automatic [Takeoff mode](../flight_modes/takeoff.md).
+Usually this is triggered from an [RC switch](../config/flight_mode.md) or ground station.
 
-Multicopter (and VTOL in multicopter mode) pilots can take off *manually* by enabling [position mode](../flight_modes/README.md#position_mc), arming the vehicle, and then raising the throttle stick above 62.5%. 
+Multicopter (and VTOL in multicopter mode) pilots can take off *manually* by enabling [position mode](../flight_modes/README.md#position_mc), arming the vehicle, and then raising the throttle stick above 62.5%.
 Above this value all controllers are enabled and the vehicle goes to the throttle level required for hovering ([MPC_THR_HOVER](../advanced_config/parameter_reference.md#MPC_THR_HOVER)).
 
-> **Note** The vehicle may disarm if you take too long to take off after arming. 
-  The timout can be tuned using [COM_DISARM_PRFLT](../advanced_config/parameter_reference.md#COM_DISARM_PRFLT).
+> **Tip** The automatic takeoff mode is highly recommended, in particular for Fixed Wing vehicles!
 
 <span></span>
-> **Tip** The automatic takeoff mode is highly recommended, in particular for Fixed Wing vehicles.
+> **Note** The vehicle may disarm if you take too long to take off after arming (tune the timeout using [COM_DISARM_PRFLT](../advanced_config/parameter_reference.md#COM_DISARM_PRFLT)).
+
+<span></span>
+> **Note** A poorly tuned vehicle may flip on takeoff.

```

Comment:
I was in two minds about adding info in this introductory topic..
-  This is where people might discover the info, and it speaks well for PX4 we handle the case.
- On the other hand, do we want people to think about flipping vehicles when looking at concepts?

Thoughts?

Reply:
Maybe don't say that "A poorly tuned vehicle may flip on takeoff" but rather that "If a failure occurs during takeoff, the failure detector will ...". So we only mention the feature.

Reply:
Thanks. I changed to 

> **Note** The [Failure Detector](../config/safety.md#failure_detector) will automatically stop the engines if there is a problem on takeoff.